packages/jupyter-ai/src/components/chat-settings.tsx

```diff
@@ -146,31 +146,20 @@ export function ChatSettings(props: ChatSettingsProps): JSX.Element {
     const newApiKeys: Record<string, string> = {};
     const lmAuth = lmProvider?.auth_strategy;
     const emAuth = emProvider?.auth_strategy;
-    if (
-      lmAuth?.type === 'env' &&
-      !server.config.api_keys.includes(lmAuth.name)
-    ) {
+    if (lmAuth?.type === 'env') {
```

Comment:
I do not follow changes around here, I will leave it to someone more familiar with this logic to review.